components/brave_rewards/resources/extension/brave_rewards/background/events/rewardsEvents.ts

```diff
@@ -4,8 +4,8 @@
 
 import rewardsPanelActions from '../actions/rewardsPanelActions'
 
-chrome.braveRewards.onWalletCreated.addListener(() => {
-  rewardsPanelActions.onWalletCreated()
+chrome.braveRewards.onWalletInitialized.addListener((result: number) => {
```

Comment:
instead of number, you can use `Rewards.Result`

Reply:
Done